README.md

```diff
@@ -17,6 +17,34 @@ As an input param it takes a json string in directus filter format and applies t
 
 ```
 
+## Queryable usage
+
+When you implement the IApiClient interface into your API client class, you are able to run linq queries
+against it like this:
+
+``` csharp
+using RuleFilterParser;
+
+IApiClient client = ...;
+
+var queryable = client.GetAsQueryable("persons");  // URL path of the API
+
+var persons = from person in queryable
```

Comment:
They both work for me. When I have simple Linq queries, I prefer the `from .. in ... select` option instead because it makes the code in my opinion better readable.

But as soon as I end up doing joins, group by operations etc. I agree with you...

Reply:
@piotrczyz Maybe someone could search for a C# tool like Pythons [`pre-commit`](https://pre-commit.com/) which performs autochecks/changes on `git commit` according to the predefined code style?

Reply:
That would be something!

Reply:
Right now, I've just applied `dotnet format` https://github.com/dotnet/format